packages/teleport/src/Databases/AddDialog/Manually/Manually.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  formatDatabaseInfo,
+  DatabaseInfo,
+} from 'teleport/services/databases/makeDatabase';
+
+export default function Manually({ user, version, onClose }) {
+  const { hostname, port } = window.document.location;
+  const host = `${hostname}:${port || '443'}`;
+
+  const [dbOptions] = useState<Option<DatabaseInfo>[]>(() =>
+    options.map(dbOption => {
+      return {
+        value: dbOption,
+        label: dbOption.title,
+      };
+    })
+  );
+
+  const [selectedDbOption, setSelectedDbOption] = useState<
+    Option<DatabaseInfo>
+  >(dbOptions[0]);
+
+  return (
+    <>
+      <DialogContent>
+        <Box mb={4}>
+          <Text bold as="span">
+            Step 1
+          </Text>
+          {' - Download Teleport package to your computer '}
+          <Box>
+            <Link href={links.getMacOS(version)} target="_blank" mr="2">
+              MacOS
+            </Link>
+            <Link href={links.getLinux64(version)} target="_blank" mr="2">
+              Linux 64-bit
+            </Link>
+            <Link href={links.getLinux32(version)} target="_blank">
+              Linux 32-bit
+            </Link>
+          </Box>
+        </Box>
+        <Box mb={4}>
+          <Text bold as="span">
+            Step 2
+          </Text>
+          {' - Login to Teleport'}
+          <TextSelectCopy
+            mt="2"
+            text={`tsh login --proxy=${host} --auth=${cfg.getAuthType()} --user=${user}`}
+          />
+        </Box>
+        <Box mb={4}>
+          <Text bold as="span">
+            Step 3
+          </Text>
+          {' - Generate a join token'}
+          <TextSelectCopy mt="2" text="tctl tokens add --type=db" />
+        </Box>
+        <Box mb={4}>
+          <Text bold as="span">
+            Step 4
+          </Text>
+          {` - Select the database type and protocol to use`}
+          <Box mt={2}>
+            <Select
+              value={selectedDbOption}
+              onChange={(o: Option<DatabaseInfo>) => setSelectedDbOption(o)}
+              options={dbOptions}
+              isSearchable={true}
+            />
+          </Box>
+        </Box>
+        <Box mb={4}>
+          <Text bold as="span">
+            Step 5
+          </Text>
+          {' - Start the Teleport agent with the following parameters'}
+          <TextSelectCopy
+            mt="2"
+            text={`${generateDbStartCmd(
+              selectedDbOption.value.type,
+              selectedDbOption.value.protocol,
+              host
+            )}`}
+          />
+        </Box>
+        <Box>
+          {`* Note: For a self-hosted Teleport version, you may need to update DNS and obtain a TLS certificate for this application.
+            Learn more about database access `}
+          <Link
+            href={'https://goteleport.com/docs/database-access/'}
+            target="_blank"
+            mr="2"
+          >
+            here
+          </Link>
```

Comment:
```suggestion
          <Link
            href={'https://goteleport.com/docs/database-access/'}
            target="_blank"
          >
            here
          </Link>
          .
```
need period after `here`